internal/provider/resource_packer_channel_test.go

```diff
@@ -0,0 +1,249 @@
+				},
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccPackerChannel_UpdateAssignedIteration(t *testing.T) {
+	resourceName := "hcp_packer_channel.production"
+	acctestChannelName := "update-test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t, map[string]bool{"aws": false, "azure": false}) },
+		ProviderFactories: providerFactories,
+		CheckDestroy: func(*terraform.State) error {
+			deleteBucket(t, acctestAlpineBucket, false)
+			return nil
+		},
+		Steps: []resource.TestStep{
+			{
+				PreConfig: func() {
+					fingerprint := "channel-update-it1"
+					upsertBucket(t, acctestAlpineBucket)
+					upsertIteration(t, acctestAlpineBucket, fingerprint)
+					itID, err := getIterationIDFromFingerPrint(t, acctestAlpineBucket, fingerprint)
+					if err != nil {
+						t.Fatal(err.Error())
+					}
+					upsertBuild(t, acctestAlpineBucket, fingerprint, itID)
+				},
+				Config: testConfig(testAccPackerChannelAssignedLatestIteration(acctestAlpineBucket, acctestChannelName)),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "author_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "created_at"),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttr(resourceName, "bucket_name", acctestAlpineBucket),
+					resource.TestCheckResourceAttr(resourceName, "name", acctestChannelName),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration_assignment.0.id"),
+				),
+			},
+			{
+				PreConfig: func() {
+					fingerprint := "channel-update-it2"
+					upsertIteration(t, acctestAlpineBucket, fingerprint)
+					itID, err := getIterationIDFromFingerPrint(t, acctestAlpineBucket, fingerprint)
+					if err != nil {
+						t.Fatal(err.Error())
+					}
+					upsertBuild(t, acctestAlpineBucket, fingerprint, itID)
+				},
+				Config: testConfig(testAccPackerChannelAssignedLatestIteration(acctestAlpineBucket, acctestChannelName)),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "author_id"),
+					resource.TestCheckResourceAttr(resourceName, "bucket_name", acctestAlpineBucket),
+					resource.TestCheckResourceAttrSet(resourceName, "created_at"),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration_assignment.0.id"),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration.0.id"),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration.0.incremental_version"),
+					resource.TestCheckResourceAttr(resourceName, "iteration.0.fingerprint", "channel-update-it2"),
+					resource.TestCheckResourceAttr(resourceName, "name", acctestChannelName),
+					resource.TestCheckResourceAttrSet(resourceName, "updated_at"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccPackerChannel_UpdateAssignedIterationWithFingerprint(t *testing.T) {
+	resourceName := "hcp_packer_channel.staging"
```

Comment:
This test is failing for me:
```
terraform-provider-hcp (nywilken/add-packer-channel-resource) ✗ (~˘▾˘)~ ✧  make testacc TESTARGS='-run=PackerChannel'                
==> Checking that code complies with gofmt requirements...
golangci-lint run --config ./golangci-config.yml 
TF_ACC=1 go test ./internal/... -v -run=PackerChannel -timeout 210m -parallel=10
testing: warning: no tests to run

=== RUN   TestAccPackerChannel
--- PASS: TestAccPackerChannel (10.50s)
=== RUN   TestAccPackerChannel_AssignedIteration
--- PASS: TestAccPackerChannel_AssignedIteration (18.21s)
=== RUN   TestAccPackerChannel_UpdateAssignedIteration
--- PASS: TestAccPackerChannel_UpdateAssignedIteration (14.24s)
=== RUN   TestAccPackerChannel_UpdateAssignedIterationWithFingerprint
    resource_packer_channel_test.go:181: Step 1/1 error: Check failed: Check 1/10 error: Not found: hcp_packer_channel.staging in [root]
--- FAIL: TestAccPackerChannel_UpdateAssignedIterationWithFingerprint (4.18s)
FAIL
FAIL	github.com/hashicorp/terraform-provider-hcp/internal/provider	47.876s
FAIL
```

Reply:
Thanks for catching this. It should be fixed now.